use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -124,35 +143,28 @@ EOF
 $ sudo clr-boot-manager update
 ```
 
-### Replace Clear Linux kernel with LTS kernel
+### Replace Clear Linux kernel with LTS kernel (Optional)
+
+If you plan to replace the host driver then the LTS kernel is necessary. This
+installs the Linux LTS kernel version `4.19` from 2018 because the 
+crypto modules are set to `m` and not `y`, which allows you to replace them 
```

Comment:
Does this imply the user will have to remain on the 4.19 kernel? Is there a security impact of this? If so, we need a **Warning:** [admonition](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#warnings-and-other-admonitions).

Reply:
The Clear Linux team backports security patches. I would think LTS implies it is maintained from a security perspective.

Reply:
I decided to remove the 4.19 section. It was only there if you want to compile and load the out of tree drivers because you can't do that on the 5.x kernel.